go-apps/meepctl/utils/helm.go

```diff
@@ -37,15 +37,14 @@ func IsHelmRelease(name string, cobraCmd *cobra.Command) (exist bool, err error)
 	if verbose {
 		fmt.Println("Cmd:", cmd.Args)
 	}
-	out, err := cmd.CombinedOutput()
+	out, err := cmd.Output()
```

Comment:
As discussed, CombinedOutput provides error and warnings that are necessary for the verbose.
I would recommend to parse the CombinedOutput to see if there is (1) a line that starts with the name and (2) that line is an exact match to the name (this excludes release that may start with the same name)

Reply:
Modified code to parse combined output and search for a line with an exact match, which is what this function should do. A function to check for a matching prefix can be added when required.